editor/inserter/menu.js

```diff
@@ -56,7 +56,7 @@ class InserterMenu extends wp.element.Component {
 	selectBlock( slug ) {
 		return () => {
 			this.props.onInsertBlock( slug );
-			this.props.onSelect();
+			this.props.closeMenu();
```

Comment:
>I think the `cloneMenu` prop should receive a nullable selected block (or slug) as a parameter (even if it's not used right now).

So consolidating to `onSelect( slug: ?string )` in place of the `onInsertBlock` and `closeMenu` combo?